support-frontend/cloud-formation/CODE-deploy.sh

```diff
@@ -0,0 +1,8 @@
+# Run this file to test cloudformation in CODE
+
+aws cloudformation deploy \
+    --profile membership \
+    --stack-name support-CODE-frontend \
+    --template-file cfn.yaml \
+    --region eu-west-1 \
+    --capabilities CAPABILITY_IAM
```

Comment:
why a script and not a cloudformation deploy through riff-raff?

Reply:
this is useful for being able to iterate quickly when developing locally

Normally we run our code in Play/webpack when running locally, but that isn't possible for cloudformation.

We were using the console to update the cloudformation, but it's a bit dangerous in case we actually choose PROD by mistake, also it's a lot of clicks.

Perhaps we can make the comment clearer though @damien-shaw-guardian ?